crates/blockchain-tree/src/blockchain_tree.rs

```diff
@@ -1600,7 +1600,6 @@ mod tests {
 
             #[cfg(not(feature = "optimism"))]
             let receipts_root = calculate_receipt_root(&receipts);
-
             #[cfg(feature = "optimism")]
```

Comment:
copy this whole test and move it to `reth-optimism-consensus`. we don't want the dep `reth-blockchain-tree` -> `reth-optimism-consensus.`

alternatively, add a test utils module to `reth-blockchain-tree`, move the test body into a new function in that module and make it generic over a type that can calculate the receipts root. that way we avoid redundant code. is also fine to open an issue for that, and just copy paste the test in op crate for now.

Reply:
Which test are you talking about like the while tests suite inside mod tests{}

Reply:
the `consecutive_reorgs` test

Reply:
ok

Reply:
Did that can you check